backend/build_dev.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+docker build -t dev .

```

Comment:
Is it the norm to have a .sh file or should we have a makefile instead?

Reply:
yes we should have a makefile.